internal/blackbox_android.py

```diff
@@ -119,18 +119,19 @@ def run_task(self, task):
             if command['command'] == 'navigate':
                 task['page_data']['URL'] = command['target']
                 activity = '{0}/{1}'.format(self.config['package'], self.config['activity'])
-                cmd = 'am start -n {0} -a android.intent.action.VIEW -d "{1}"'.format(activity, \
-                        command['target'].replace('"', '%22'))
+                cmd = 'am start -n {0} -a android.intent.action.VIEW \
+                       -d "{1}"'.format(activity,
+                                        command['target'].replace('"', '%22'))
                 local_intent = os.path.join(task['dir'], 'wpt_intent.sh')
                 remote_intent = '/data/local/tmp/wpt_intent.sh'
                 self.adb.shell(['rm', remote_intent])
-                with open(local_intent, 'wb') as f_out:
+            with open(local_intent, 'wb') as f_out:
```

Comment:
Why did this move out if the if block?

Reply:
Sorry, over-eager "fix" of a spurious indentation warning which actually arose from another PEP8 fix.

Leaving this last warning in unless you want me to delete that variable and its assignment:

```
diff --git a/internal/blackbox_android.py b/internal/blackbox_android.py
index 0ceaa60..8d01278 100644
--- a/internal/blackbox_android.py
+++ b/internal/blackbox_android.py
@@ -125,7 +125,7 @@ class BlackBoxAndroid(AndroidBrowser):
                 local_intent = os.path.join(task['dir'], 'wpt_intent.sh')
                 remote_intent = '/data/local/tmp/wpt_intent.sh'
                 self.adb.shell(['rm', remote_intent])
-            with open(local_intent, 'wb') as f_out:
+                with open(local_intent, 'wb') as f_out:
                     f_out.write(cmd)
             if self.adb.adb(['push', local_intent, remote_intent]):
                 os.remove(local_intent)
Stephens-MacBook-Pro:internal stephendonner$ flake8 blackbox_android.py 
blackbox_android.py:180:13: F841 local variable 'modified' is assigned to but never used
````